extension.css

```diff
@@ -279,107 +272,102 @@ button.browser-style.expander {
   width: 24px;
 }
 
-/* Interactive States */
-button.browser-style:hover:not(.pressed):not(.disabled):not(.focused),
-select.browser-style:hover:not(.pressed):not(.disabled):not(.focused) {
-  background-color: #ebebeb;
-  border: 1px solid #b1b1b1;
-}
-
-button.browser-style:hover:active:not(.hover):not(.disabled):not(.focused),
-select.browser-style:hover:active:not(.hover):not(.disabled):not(.focused) {
-  background-color: #d4d4d4;
-  border: 1px solid #858585;
-}
-
-button.browser-style.default:hover:not(.pressed):not(.disabled):not(.focused) {
-  background-color: #0670cc;
-  border-color: #005bab;
-}
-
-button.browser-style.default:hover:active:not(.hover):not(.disabled):not(.focused) {
-  background-color: #005bab;
-  border-color: #004480;
-}
-
-button.browser-style:focus:not(.disabled) {
-  border-color: #fff !important;
-  box-shadow: 0 0 0 2px rgba(97, 181, 255, 0.75);
-}
-
 /* Fields */
 .browser-style > input[type="text"],
 textarea.browser-style {
   background-color: #fff;
-  border: 1px solid #b1b1b1;
-  box-shadow: 0 0 0 0 rgba(97, 181, 255, 0);
-  font: caption;
+  border: 1px solid var(--grey-20);
+  border-radius: 2px;
+  font: inherit;
+  margin-bottom: 6px;
   padding: 0 6px 0;
   transition-duration: 250ms;
   transition-property: box-shadow;
+  transition-timing-function: cubic-bezier(.07,.95,0,1);
```

Comment:
I'm **shocked** we don't have this as a token-variable yet! Can we fix that, please?